auth_jwt/models/auth_jwt_validator.py

```diff
@@ -114,7 +114,7 @@ def _decode(self, token):
                 header = jwt.get_unverified_header(token)
             except Exception as e:
                 _logger.info("Invalid token: %s", e)
-                raise UnauthorizedInvalidToken()
+                raise UnauthorizedInvalidToken() from e
```

Comment:
This change should not be here yet, as you are just setting the modules as installable=False